index.html

```diff
@@ -48,8 +48,13 @@
 
     <section id="hackathon" class="content-section text-center">
       <div class="row">
-        <div class="col-lg-8 col-lg-offset-2">
-          <h2>Hacakthon</h2>
+        <div class="col-lg-8 col-lg-offset-2" style= color:black; >
```

Comment:
Please use css. Do not use inline styles.
Example: https://github.com/SpectraHackathon/spectra-revamp/blob/master/team/frontend/main.css